pipelines/unittests/Jenkinsfile

```diff
@@ -106,15 +124,36 @@ pipeline {
 //___________________________________________________________________________________________
 // -------------------------------- CLEAN DATABASE ------------------------------------------
         stage('Clean Database') {
+            when {
+                expression {
+                    env.STATUSBUILD == "1"
```

Comment:
Couldn't there be a situation where the build fails, but the database was created?

For example, while executing the install task, the update.database fails, but the database was already created before that